web/javascripts/functions.js

```diff
@@ -0,0 +1,60 @@
+function updateCircle() {
+  const select = document.getElementById('state');
+  const selectedValue = select.value;
+  const circle = document.getElementById('circle');
+
+  switch (selectedValue) {
+      case 'ongoing':
+          circle.className = 'w-2 h-2 bg-orange-500 rounded-full mr-2';
+          break;
+      case 'completed':
+          circle.className = 'w-2 h-2 bg-green-500 rounded-full mr-2';
+          break;
+      case 'pending':
+          circle.className = 'w-2 h-2 bg-gray-500 rounded-full mr-2';
+          break;
+  }
+}
+
+// Message is the message to display in the modal, title is the title of the modal, buttonText is the text of the button, redirect is a boolean to redirect to home or not
+
+function showModal(title, message, buttonText, redirect) {
+  const modal = document.getElementById('modal');
+  const modalBody = document.getElementById('modalBody');
+  const modalTitle = document.getElementById('modalTitle');
+  const closeModalButton = document.getElementById('closeModal');
+  const closeBtn = document.getElementById('closeBtn');
+
+  const setModalContent = () => {
+    modalTitle.textContent = title;
+    modalBody.textContent = message;
+    closeBtn.textContent = buttonText;
+  };
+
+  const closeModal = () => {
+    modal.classList.add('hidden');
+    if(redirect) window.location.href='/';
```

Comment:
Maybe it would be interesting to leave redirection open, so that when modal is shown in `/create` it redirects to `view/:id`, but when it is shown when triggered by `/delete` (we're yet to see how to add multiple ids there 😅) it redirects to `/`. 

@AlexP953, what do you think?

Reply:
Sounds good! I've changed the code so that the new route can be sent by parameter. I want to look into how to redirect to view/:id but for now it can stay like this to continue moving forward. Thanks!